src/me/ryanhamshire/GriefPrevention/AutoExtendClaimTask.java

```diff
@@ -84,11 +84,11 @@ private int getLowestBuiltY()
                     {
                         for(int z = 0; z < 16; z++)
                         {
-                            int blockType = chunk.getBlockTypeId(x, y, z);
```

Comment:
The dangers of oblivious ctrl-f usage! Context will help you see why this is used here!

Reply:
What’s wrong with this? #getBlockTypeId’s been removed so i moved to #getBlockType(string)

Reply:
This is in the try-catch block. For the NoSuchMethod exception. (For older 1.12.2 spigot builds that don't include this new chunk API).

Reply:
https://github.com/TechFortress/GriefPrevention/commit/9c5c7523a29095d1485d5b08b9d2c10915d6c386

Reply:
Yea i know, but the new 18w02a API does not have the Material#getBlockTypeId method; as ID’s don’t exist anymore. That’s the reason why i moved over to a string name of the block and used the `Material#getBlockType(string material)` method

Reply:
Then this try-catch would be removed altogether then, rather than effectively repeating the above.